fantasy_football_project/settings.py

```diff
@@ -26,12 +26,61 @@
 
 # SECURITY WARNING: keep the secret key used in production secret!
 load_dotenv()
+
 SECRET_KEY = os.environ.get("DJANGO_SECRET_KEY")
 TANK_API_KEY = os.environ.get("TANK_API_KEY")
 TANK_API_ENDPOINT = "tank01-nfl-live-in-game-real-time-statistics-nfl.p.rapidapi.com"
 
 DEBUG = False
 
+# ---------------------------------------------
+# ---- Auth -----
+# - CORS + CSRF + Cookies settings
+# TODO: this works in local dev right now
+# TODO: need to nail down how itll work in Prod
+# ---------------------------------------------
+CORS_ORIGIN_ALLOW_ALL  = True 
```

Comment:
We'll probably want to review each of these. We'll definitely want `CORS_ORIGIN_ALLOW_ALL = False`
`CORS_ALLOW_ALL_ORIGINS = False`
I don't know what a lot of these are so we can go through them a bit and figure it out. We can move all of these to the dev section (line 162+) so it definitely works in dev and then fiddle with it for prod